packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -89,7 +117,7 @@ const ReleasesGrid = ({ sectionTitle, releases = [], isError = false }: Releases
 
   return (
     <Grid gap={4}>
-      {releases.map(({ id, name, actions, scheduledAt }) => (
+      {releases.map(({ id, name, actions, scheduledAt, status }) => (
         <GridItem col={3} s={6} xs={12} key={id}>
           <LinkCard href={`content-releases/${id}`} isExternal={false}>
             <Flex
```

Comment:
We gonna need to change this component to make it look more similar to desings. Right now we have this "Flex" parent with the 3 components (Title, subtitle & badge) as child with a gap between them. Probably we would need to have this Flex Parent and then one "Header" child with the title and subtitle and then the badge. Something like this:

Flex
-- Header (Flex)
---- Title
---- Subtitle
-- Badge

App:
![Screenshot 2024-02-29 at 09 36 33](https://github.com/strapi/strapi/assets/23050694/61832413-8543-46ed-8f5e-602a5163e130)
Design:
![Screenshot 2024-02-29 at 09 36 46](https://github.com/strapi/strapi/assets/23050694/1dba808d-7b30-426b-b223-2c20185bf8ae)

(There is too much space between title & subtitle)